src/main/java/de/retest/web/selenium/By.java

```diff
@@ -0,0 +1,9 @@
+package de.retest.web.selenium;
+
+public abstract class By extends org.openqa.selenium.By {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this class. ([squid:S2176](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS2176))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=77&issues=AWaHMZUglB6-zLfzofF9&open=AWaHMZUglB6-zLfzofF9)

[ISSUE_KEY]: AWaHMZUglB6-zLfzofF9